divi/src/test/rpc_wallet_tests.cpp

```diff
@@ -7,7 +7,7 @@
 
 #include "base58.h"
 #include "wallet.h"
-#include <main.h>
+//#include <main.h>
```

Comment:
As Yoda said: There is no comment, only delete

Reply:
done